packages/Python/lldbsuite/test/lldbinline.py

```diff
@@ -212,6 +212,24 @@ def ApplyDecoratorsToFunction(func, decorators):
         tmp = decorators(tmp)
     return tmp
 
+def isEnabled(debug_flavor, target_platform, configuration, decorators):
+    # If the platform is unsupported, skip the test.
+    if not test_categories.is_supported_on_platform(debug_flavor,
+            target_platform, configuration.compiler):
+        return False
+
+    # If the debug flavor is skipped, skip the test.
+    if debug_flavor in configuration.skipCategories:
+        return False
+
+    # If there are any decorators, and we're in swift PR testing mode, skip the
```

Comment:
Can you explain why this is necessary? I understand that we do not care, but I don't like this localized exception which makes looking at an inline test ever more opaque than it already is.

Reply:
It's necessary to keep expectedFail tests out of swift PR testing (I'll update the comment to reflect this). Happy to do this some other way if possible!

Reply:
Fair enough. This is also a problem for non-inline tests, but there we would just remove the `swiftpr` category if a test is XFAIL'ed/skipped while here we can't do that, hence why we need this. Am I understanding that correctly?

Reply:
Exactly.

Reply:
Alright, makes sense, maybe worth mention that it's more a matter of "can't" rather than "don't want".

Reply:
Does the new comment look any better?

Reply:
Yes, thanks!